packages/firebase_auth/firebase_auth/lib/src/firebase_auth.dart

```diff
@@ -85,7 +85,7 @@ class FirebaseAuth extends FirebasePluginPlatform {
     String mappedOrigin = origin;
 
     // Android considers localhost as 10.0.2.2 - automatically handle this for users.
-    if (defaultTargetPlatform == TargetPlatform.android) {
+    if (defaultTargetPlatform == TargetPlatform.android && !kIsWeb) {
```

Comment:
That is bizarre - I *know* it worked, I tested it! This seems related to that issue recently where you switched CI to ubuntu and it thinks it is android now?

Reply:
Yep, had to do this on Firestore as well a few PRs back, Flutter says the platform is Android when running on Linux, so the logic here made Web point to `10.0.2.2` which blew up.

Reply:
That's exactly where the log message discussed with @russellwheatley for storage emulator could save a lot of heartache in the future. It's a really subtle failure and kind of inscrutable to a dev if we remap in error and connection fails. Could maybe add similar log message here just in case

https://github.com/invertase/react-native-firebase/blob/589fcb07d78016841287e818291ebb9bab0d4e89/packages/auth/lib/index.js#L364-L376

Reply:
```javascript
        console.log(
          'Mapping auth host "127.0.0.1" to "10.0.2.2" for android emulators. Use real IP on real devices.',
        );
```